contents/docs/integrate/_snippets/install-flutter.mdx

```diff
@@ -6,63 +6,116 @@ Set your PostHog API key and change the automatic event tracking on if you wish
 
 Remember that the application lifecycle events won't have any special context set for you by the time it is initialized. If you are using a self-hosted instance of PostHog you will need to have the public hostname or IP for your instance as well.
 
-#### Android
+To start, add `posthog_flutter` to your `pubspec.yaml`:
 
-```xml file=AndroidManifest.xml
-<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.posthog.posthog_flutter_example">
+```yaml file=pubspec.yaml
+# rest of your code
+
+dependencies:
+  flutter:
+    sdk: flutter
+  posthog_flutter: ^4.0.1
+
+# rest of your code
+```
+
+Then complete the set up for each platform:
+
+#### Android setup
+
+For Android, add your PostHog configuration to your `AndroidManifest.xml` file located in the `android/app/src/main`:
+
+```xml file=android/app/src/main/AndroidManifest.xml
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="your.package.name">
     <application>
-        <activity>
-            [...]
-        </activity>
+        <!-- ... other configuration ... -->
         <meta-data android:name="com.posthog.posthog.API_KEY" android:value="<ph_project_api_key>" />
-        <!-- usually 'https://app.posthog.com' or 'https://eu.posthog.com' -->
         <meta-data android:name="com.posthog.posthog.POSTHOG_HOST" android:value="<ph_instance_address>" />
         <meta-data android:name="com.posthog.posthog.TRACK_APPLICATION_LIFECYCLE_EVENTS" android:value="true" />
         <meta-data android:name="com.posthog.posthog.DEBUG" android:value="true" />
     </application>
 </manifest>
 ```
 
-#### iOS/macOS
+/Users/lior/Desktop/flutter_ab_tests_2/flutter_ab_tests_2/
 
-```xml file=Info.plist
+You'll also need to update the minimum Android SDK version to `21` in `android/app/build.gradle`:
+
+```js file=android/app/build.gradle
+// rest of your config
+
+    defaultConfig {
+      minSdkVersion 21
+      // rest of your config
+    }
+
+// rest of your config
+```
+
+You may also need to ensure you're running on the latest Kotlin version. To do this, go to the [Kotlin website](https://kotlinlang.org/docs/releases.html#release-details) to find the latest version number. Then set this version number in the `build.gradle` in the root of your `android` directory (and not the `build.gradle` located in `android/app`):
+
+```js file=android/build.gradle
+buildscript {
+    ext.kotlin_version = '1.9.22' // use the latest Kotlin version number here
```

Comment:
Since 1.8.x would be fine already it's ok to keep 1.9.x
The reason is not us enforcing the newest Kotlib version but some basic Android dependencies such as the `lifecycle` we rely on.

Reply:
the default `kotlin_version`should work but its ok to keep it.